Formula/c/cppman.rb

```diff
@@ -18,6 +18,7 @@ class Cppman < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "bd1d8e0b3eb0e6cf71be41fd1b3de80029f5f7bc8737c91b59433fb378d574bd"
   end
 
+  depends_on "groff"
```

Comment:
groff is present on macOS before Ventura, so this should be:

```suggestion
  on_system :linux, macos: :ventura_or_newer do
    depends_on "groff"
  end
```

Reply:
good to know, I've amended the commit